### ISSUE TITLE
All process crash if one point doesnt fit properly

### ISSUE THREAD

**Issue by rdemets** [fmi-faim/napari-psf-analysis#61]
Hi. 

Several point under a single Issues, but hopefully it would work. 

I second the comments on the metadata filling automatically the GUI, even though I foresee some issues depending on the image type. 

I also would like to ask about recommendation of voxel size. 
I tried several test by using the original image of 20x20x100nm, then downsampled the image using Fiji to 40x40x100 and I found the fit way more convincing on the second stack. I was wondering if you have recommendation to what the ratio between axial and lateral step size should be to get a good fit. 

More related to the title of the issue, I found it frustrating if I create 20 points  and the whole process is cancelled because one fit didn't work (XYFitRecord failed because it got a negative input value). It would be good if it was possible to catch those errors and skip to the next point if possible. 

Also, would there be a way to get a tick box asking if we want to have an average PSF with SD to see the homogeneity of the FOV. 

Lastly, regarding the icon creation, I followed the twitter thread when it came out but I found a much simpler way, which consist in creating a Shortcut on the desktop with a location such as : 

C:\Anaconda3\Scripts\activate.bat activate C:\Users\username\.conda\envs\psf-analysis & napari --with psf-analysis
This should allow to open napari with the plugin open directly in the viewer without having to open the anaconda prompt.

Please let me know if there is a way or if you are working on developing those things, otherwise I might just do it on my own. 
Regards. Richard.

**Comment by tibuch** [comment]
Hi @rdemets,

Thank you for reaching out and reporting these issues. I will split them up into single issues for ease of tracking. 

I think all of them should be addressed either via proper documentation or bug fixes. 

Please feel invited to open pull requests to fix any issue. This would be of great help!

Kind regards

**Comment by MaxusTheOne** [comment]
Hi tibuch

I have been developing a branch of your plugin for some months now, and it includes some of the features like getting data from metadata (.czi only for now) catching exceptions, an average psf from all other psf images. 
It also includes a lot of other changes, like a bead estimator or image quality analysis.
If you want to check it out: [psf-analysis-CFIM](https://github.com/MaxusTheOne/napari-psf-analysis-CFIM-edition)

I'm very new with things like, license, crediting and other stuff that would be useful :3
If there are any concerns or changes you would like regarding crediting I would love to have them

Mvh Markus Max

**Comment by imagejan** [comment]
Hi @MaxusTheOne,

thanks for pointing to your improvements in your forked project. As long as those are of general interest, would you be willing to contribute them as a pull request to this repository?

**Comment by MaxusTheOne** [comment]
Of course! But I don't know how easy the individual changes would be to implement.
I have changed a bunch of the general structure. More than 60 commits, and I think thousands of lines of changes

Id be happy to set up a pull request or similar if you find it usefull
